packages/runtime/container-runtime/src/summarizerTypes.ts

```diff
@@ -221,3 +225,56 @@ export interface ISummarizer
         options: Omit<ISubmitSummaryOptions, "summaryLogger">,
     ): OnDemandSummarizeResult;
 }
+
+/** Data about an attempt to summarize. */
+export interface ISummaryAttempt {
+    /** Reference sequence number when summary was generated or attempted */
+    readonly refSequenceNumber: number;
+
+    /** Time of summary attempt after it was sent or attempted */
+    readonly summaryTime: number;
+
+    /** Sequence number of summary op */
+    summarySequenceNumber?: number;
+}
+
+/** Data relevant for summary heuristics. */
+export interface ISummarizeHeuristicData {
+    /** Latest received op sequence number */
+    lastOpSequenceNumber: number;
+
+    /** Most recent summary attempt from this client */
+    readonly lastAttempt: ISummaryAttempt;
+
+    /** Most recent summary that received an ack */
+    readonly lastSuccessfulSummary: Readonly<ISummaryAttempt>;
```

Comment:
nit: maybe `lastSuccessfulAttempt` since it is an attempt that was successful :-)

Reply:
Yeah that was my first draft, but then I realized that during first load, it actually is based on the last ack we loaded from... 🤷‍♂️

Reply:
Maybe the name `ISummaryAttempt` is not apt? Can we think of a better name for it?